source/features/tags-on-commits-list.tsx

```diff
@@ -90,7 +90,13 @@ async function init(): Promise<void | false> {
 	await expectToken();
 	const cacheKey = `tags:${getRepo()!.nameWithOwner}`;
 
-	const commitsOnPage = $$('[data-testid="commit-row-item"]');
+	let commitsOnPage = $$optional('[data-testid="commit-row-item"]');
+	if (commitsOnPage.length === 0) {
+		// Try waiting a bit longer
+		// https://github.com/refined-github/refined-github/issues/7954
+		await delay(1000);
+		commitsOnPage = $$('[data-testid="commit-row-item"]');
+	}
```

Comment:
- Normal users: unaffected
- GHE users: the feature retries after 1 second
- Future normal users of broken feature: error appears after 1 second, but is explicitly `Expected elements not found: [data-testid="commit-row-item"]` rather than `Expected element not found because the base is specified but null` on the wrong line